src/index.html

```diff
@@ -6,14 +6,403 @@
+    
+            <div class="section__content">
+              <h2 class="section__title title">Ayurvedic Tooth Powder</h2>
+    
+              <div class="section__description">
+                <p class="text">Kaolin</p>
+                <p class="text">Sivak</p>
+                <p class="text">Calcium</p>
+                <p class="text">Licorice root</p>
+                <p class="text">Turmeric</p>
+              </div>
+    
+              <a href="#shop" class="section__link link-hover">
+                LEARN MORE
+              </a>
+            </div>
+    
+            <div class="section__images">
+              <div class="powder__image">
+                <img src="images/powder/tooth.svg" alt="Ayurvedic Tooth Powder" />
+              </div>
+            </div>
+          </div>
+        </section>
+    
+        <section class="page__info info">
+          <div class="info__container">
+            <div class="info__item">
+              <img src="images/info/turtle.png" alt="turtle" />
+              <div class="info__text">No synthetics and microplastics</div>
+            </div>
+    
+            <div class="info__item">
+              <img src="./images/info/icon.png" alt="icon" />
+              <div class="info__text">
+                product packaging can be recycled or reused
+              </div>
+            </div>
+    
+            <div class="info__item">
+              <img src="./images/info/leafe.png" alt="leafe" />
+              <div class="info__text">only natural ingredients</div>
+            </div>
+          </div>
+        </section>
+    
+        <section class="page__contacts contacts section">
+          <div id="contacts" class="section__container">
+            <div class="decore">help</div>
+    
+            <div class="section__content">
+              <h2 class="section__title title">contact us</h2>
+    
+              <div class="section__description">
+                <p class="text">
+                  We will help you choose a product that suits you and answer
+                  your other questions
+                </p>
+              </div>
+    
+              <form
+                action="#" 
+                method="post" 
+                class="section__form form" 
+                onsubmit="this.reset()"
+              >
+                <input
+                  type="text" 
+                  name="name" 
+                  placeholder="Your name" 
+                  required pattern="\w+" 
+                  class="form__input input"
+                />
+    
+                <input
+                  type="email"
+                  name="email" 
+                  placeholder="e-mail" 
+                  required class="form__input input"
+                />
+    
+                <input
+                  type="tel" 
+                  name="number" 
+                  placeholder="phone number"
+                  required pattern="^\+380\d{9}$"
```

Comment:
"required" should be in one line, and "pattern" in other.
Just identation fixing